bindings/python/src/lib.rs

```diff
@@ -141,13 +145,14 @@ fn format_pyerr(err: od::Error) -> PyErr {
     use od::ErrorKind::*;
     match err.kind() {
         NotFound => PyFileNotFoundError::new_err(err.to_string()),
-        _ => PyRuntimeError::new_err(err.to_string()),
+        _ => OpenDalError::new_err(err.to_string()),
     }
 }
 
 #[pymodule]
-fn opendal(_py: Python, m: &PyModule) -> PyResult<()> {
+fn opendal(py: Python, m: &PyModule) -> PyResult<()> {
     m.add_class::<Operator>()?;
     m.add_class::<AsyncOperator>()?;
+    m.add("OpenDalError", py.get_type::<OpenDalError>())?;
```

Comment:
I think we should expose this type as `Error`? So users may refer them as `opendal.Error`?